app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
+    </style>
+
+    <style name="button_style">
+        <item name="android:layout_marginBottom">@dimen/margin_between_button_xsmall</item>
+    </style>
+
+    <style name="icon_style">
+        <item name="android:layout_marginStart">@dimen/margin_start_icon_small</item>
+    </style>
+
+    <style name="title_style">
+        <item name="android:gravity">center</item>
+        <item name="android:textColor">@color/blue_result</item>
+        <item name="android:textSize">@dimen/text_size_title_xbig</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="result_style">
+        <item name="android:padding">@dimen/padding_result_small</item>
+        <item name="android:textStyle">bold</item>
+        <item name="android:textColor">@color/white_background</item>
+        <item name="android:background">@color/blue_result</item>
+        <item name="android:gravity">end|bottom</item>
+        <item name="android:textSize">@dimen/text_size_result_xbig</item>
+    </style>
+
+</resources>
```

Comment:
to avoid this error add a new line after at the end of the file

Reply:
Solved!